app/views/robots/index.text.erb

```diff
@@ -0,0 +1,67 @@
+# See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
+#
+# To ban all spiders from the entire site uncomment the next two lines:
+# Disallow: /
+
+User-agent: *
+Disallow: /users/
+Disallow: /comments/
+
+Disallow: /*?*locale
+Disallow: /*?*order
+Disallow: /*?*search
+Disallow: /*?*locale-switcher
+Disallow: /*?*filter
+Disallow: user_id
+
+# Allow search by category
```

Comment:
All `Allow` conditions on this file are specific for Madrid's fork and shouldn't include on CONSUL. 😌